assignments/closure.js

```diff
@@ -1,14 +1,27 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+const first = 'Jacob';
 
+function name (){
```

Comment:
closure is a really tough concept to explain.  Your example, is more an example of function scoping where the inner function has access to the inner and outer variables.

A closure is the assigning of said inner scope to a variable for access later.  Generally speaking, most closures also return a function so that they can manipulate that inner variable like the counter below.

Reply:
So a better example would be this yes? Because the 

``` const hey = (first, last) => {
   let full = 'Hi, My name is ';
   function fullname () {
     return full + first + ' ' + last + '.'; 
   }

   return fullname();
 }

 console.log(hey('Jacob', 'Williams')) // logs out Hi, My name is Jacob Williams```